### PR TITLE
[PLUGIN-1667] GCS Move Build Fix

### PR DESCRIPTION
**Pull request by bharatgulati**
JIRA: [PLUGIN-1667](https://cdap.atlassian.net/browse/PLUGIN-1667)

[PLUGIN-1667]: https://cdap.atlassian.net/browse/PLUGIN-1667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/e2e-test/features/gcsmove/GCSDoneFileMarker.feature [hunk 1]
```diff
@@ -35,10 +35,9 @@ Feature:GCSDoneFileMarker - Validate GCSDoneFileMarker scenarios with GCSMove pi
     When Select plugin: "GCS Move" from the plugins list as: "Conditions and Actions"
     When Navigate to the properties page of plugin: "GCS Move"
     Then Enter GCSMove property projectId "projectId"
-    Then Enter GCSMove property source path
+    Then Enter input plugin property: "sourcePath" with value: "wrongSourcePath"
     Then Enter GCSMove property destination path
     Then Select GCSMove property move all subdirectories as "true"
-    Then Enter wrong value for GCSMove property service account file path
```
Comment (Aryan-Verma) [inline src/e2e-test/features/gcsmove/GCSDoneFileMarker.feature:41]: Please also remove the stepsdesign method for the same

File: src/e2e-test/resources/pluginParameters.properties [hunk 2]
```diff
@@ -1,6 +1,7 @@
 projectId=cdf-athena
 datasetprojectId=cdf-athena
 dataset=bq_automation
+wrongSourcePath=gs://abcdefgh
```
Comment (Aryan-Verma) [inline src/e2e-test/resources/pluginParameters.properties]: would be better to use a random string (of same length or more)

Comment (bharatgulati) [reply src/e2e-test/resources/pluginParameters.properties]: Done. Using the Value of 'wrongSourcePath' with the same bucket length.

File: src/e2e-test/features/gcsmove/GCSDoneFileMarker.feature (hunk 1 above)
Comment (bharatgulati) [reply src/e2e-test/features/gcsmove/GCSDoneFileMarker.feature:41]: Removed.

**Comment by bharatgulati** [comment]
Fix of GCS Done File Marker plugin e2e scenario. We are expecting the pipeline to fail but the pipeline instead succeeds when we are using service account json. (https://cdap.atlassian.net/browse/PLUGIN-1667)

**Review by Aryan-Verma — APPROVED** [review]

**Review by Aryan-Verma — APPROVED** [review]

**Pull request merged**